awx/main/tasks.py

```diff
@@ -904,6 +904,20 @@ def build_execution_environment_params(self, instance):
             "container_options": ['--user=root'],
         }
 
+        if instance.execution_environment.credential:
+            path = self.build_private_data_dir(instance)
```

Comment:
I don't _think_ we want to re-build the private data directory here, but rather we want to reference the existing one that should have gotten created earlier in the call stack.

Also this seems like it would put this file directly in the private data directory, but what we actually want is to put it in the `pdd_wrapper` directory.

Reply:
ope, I'll add that post-lunch